data/inference_sem_seg_ci_csv.csv

```diff
@@ -1,2 +1,2 @@
-./data/22978945_15.tif,
-./data/23429155_15.tif,
+./data/22978945_15.tif,,,
+./data/23429155_15.tif,,./data/massachusetts_buildings.gpkg,properties/class
```

Comment:
the gpkg is added because of the benchmarking functionality included in inference.py. inference.py now handles individual images with or without gpkg for benchmarking. The README.md has been updated which should help.